ocis-pkg/x/io/fsx/sub.go

```diff
@@ -0,0 +1,20 @@
+package fsx
+
+import (
+	"io/fs"
+	"os"
+
+	"github.com/owncloud/ocis/v2/ocis-pkg/log"
+)
+
+// MustSub logic is the same as fs.Sub, but it does not return an error,
+// but logs and exits the process if the sub filesystem cannot be loaded.
+func MustSub(fsys fs.FS, dir string) fs.FS {
```

Comment:
Nitpicking: Name is confusing, `MustLoadSubtree`

Reply:
i agree, sub sounds strange, but i wanted to keep the core lib name as it is and extend it with `MUST` to make it clear, if there is an error, the code fails and Exit.

`fs.Sub` -> https://github.com/golang/go/blob/0784fd1b2f6050b8fb91c88c1d02107258a1651a/src/io/fs/sub.go#L17